cnnimg.py

```diff
@@ -31,10 +31,7 @@
 import Image as img
 import numpy as np
 import os.path
-import warnings
 
-#Ignore warnings
```

Comment:
Well I didn't get any warnings myself, funnily enough! If you say they're intermittent, though, it might be better to display them and then figure out the reasons for their popping up. But I guess that's rather stylistic.